public/images/hubble-captures-vivid-auroras-in-jupiters-atmosphere_28000029525_o~orig.md

```
@@ -0,0 +1,35 @@
+# Hubble Captures Vivid Auroras in Jupiter’s Atmosphere
+
+**NASA ID:** `hubble-captures-vivid-auroras-in-jupiters-atmosphere_28000029525_o`
+
+Astronomers are using the NASA/ESA Hubble Space Telescope to study auroras — stunning light
+shows in a planet’s atmosphere — on the poles of the largest planet in the solar system, 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)